windows/internal/build_magma.bat

```diff
@@ -0,0 +1,92 @@
+@setlocal
+
+set MAGMA_VERSION=2.5.4
+
+set CUVER_NODOT=%CUDA_VERSION%
+set CUVER=%CUVER_NODOT:~0,-1%.%CUVER_NODOT:~-1,1%
+
+set CONFIG_LOWERCASE=%CONFIG:D=d%
+set CONFIG_LOWERCASE=%CONFIG_LOWERCASE:R=r%
+set CONFIG_LOWERCASE=%CONFIG_LOWERCASE:M=m%
+
+echo Building for configuration: %CONFIG_LOWERCASE%, %CUVER%
+
+:: Download Ninja
+curl -k https://s3.amazonaws.com/ossci-windows/ninja_1.8.2.exe --output C:\Tools\ninja.exe
```

Comment:
What? It doesn't have ninja installed in GHA?

Reply:
BTW, you'll need error code checking here. `if errorlevel 1 exit /b 1`

Reply:
Yea, I ran without ninja and it failed